Description: fix FTBFS on Debian GNU/kFreeBSD architectures
Author: Paul Wise <pabs@debian.org>
Forwarded: yes, <softcoder> will apply to svn shortly
Applied-Upstream: http://megaglest.svn.sourceforge.net/viewvc/megaglest?view=revision&revision=3022
--- a/source/shared_lib/sources/platform/posix/socket.cpp
+++ b/source/shared_lib/sources/platform/posix/socket.cpp
@@ -313,7 +313,7 @@
 //	class Socket
 // ===============================================
 
-#if defined(__FreeBSD__) || defined(BSD) || defined(__APPLE__) || defined(__linux__)
+#if defined(__FreeBSD__) || defined(__FreeBSD_kernel__) || defined(BSD) || defined(__APPLE__) || defined(__linux__)
 # define USE_GETIFADDRS 1
 # include <ifaddrs.h>
 #endif
--- a/source/glest_map_editor/CMakeLists.txt
+++ b/source/glest_map_editor/CMakeLists.txt
@@ -19,7 +19,7 @@
 		# It was noticed that when using MinGW gcc it is essential that 'core' is mentioned before 'base'.
 	
 		IF(${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
-		       SET(wxWidgets_CONFIG_OPTIONS --prefix=/usr/local)
+		       SET(wxWidgets_CONFIG_OPTIONS)
 		       MESSAGE(STATUS "Detected FreeBSD.....")
 	       ENDIF()
 
